### PR TITLE
Feature pre release integration - to avoid conflicts

### PR DESCRIPTION
**Pull request by Vivek205**
Needed @Habush 's verification

### CODE REVIEW

**Review by anandrgitnirman — APPROVED** [review]
approving as these dont clash with the paypal  / metamask component

**Comment by Habush** [comment]
@yisehak-awm take a look at this please.

File: src/assets/thirdPartyServices/mozi/gene_annotation_service/visualizer/index.js [hunk 1]
```diff
@@ -195,149 +195,137 @@ const Visualizer = props => {
...
-            .nodes()
-            .difference(filteredElements)
-            .style({ opacity: 0.1 })
-        );
-        cy.edges()
+  }, [cy_wrapper]);
+
+  useEffect(() => {
+    cy && toggleAnnotationVisibility(visibleAnnotations);
+  }, [visibleAnnotations, visibleNodeTypes, visibleLinkTypes, cy, toggleAnnotationVisibility]);
```
Comment (yisehak-awm) [inline src/assets/thirdPartyServices/mozi/gene_annotation_service/visualizer/index.js:202]: Why is the toggleAnnotationVisibility included here?

File: src/assets/thirdPartyServices/mozi/gene_annotation_service/visualizer/index.js [hunk 2]
```diff
@@ -195,149 +195,137 @@ const Visualizer = props => {
...
-  );
+            show: true,
+          },
+        ],
+        menuItemClasses: ["context-menu-item"],
+        contextMenuClasses: ["context-menu"],
+      };
+      setContextMenu(cy.contextMenus(options));
+    }
+  }, [MLLLayout, addToFilter, cy, removeFromFilter]);
```
Comment (yisehak-awm) [inline src/assets/thirdPartyServices/mozi/gene_annotation_service/visualizer/index.js:304]: @Vivek205 can you explain why functions are included in the array?

**Pull request closed without merging**